django_x509/tests/test_ca.py

```diff
@@ -666,3 +666,11 @@ def test_renew(self):
         self.assertNotEqual(old_cert2_key, cert2.private_key)
         self.assertLess(old_cert2_end, cert2.validity_end)
         self.assertNotEqual(old_cert2_serial_number, cert2.serial_number)
+
+    def test_ca_common_name_length(self):
+        common_name = (
+            'this is a very very very very very very'
+            ' very very very very very very long name'
+        )
+        with self.assertRaises(ValidationError):
+            self._create_ca(common_name=common_name)
```

Comment:
Thanks for adding the test!
In this case it was not absolutely necessary, because we can rely on the fact that django itself is tested for this, but a few lines of extra tests are better than lack of tests, so we can keep it, but we need to make sure this test is robust:
you're checking only that a `ValidationError` is raised, but it may be raised for different reasons. For example, a contributor may change the code in a way that another `ValidationError` is raised by this function, while the error you expect is not raised, in that case this test would pass anyway, that's why robust tests are more specific in the expected outcome.

In these cases we always have to check the contents of the `ValidationError`, like its done in other openwisp modules:, see here: https://github.com/openwisp/openwisp-controller/pull/266/files#diff-3fef99a26498e5609812504f227b4004R224-R230